modules/sdk-unified-wallet/src/ecdsa-unified-wallet/ecdsaUnifiedWallets.ts

```diff
@@ -1,16 +1,22 @@
-import { GenerateWalletOptions, BitGoBase, KeychainsTriplet, SupplementGenerateWalletOptions } from '@bitgo/sdk-core';
-import { UnifiedWalletKeychains } from '../unifiedWalletKeychains';
+import {
+  GenerateWalletOptions,
+  BitGoBase,
+  KeychainsTriplet,
+  SupplementGenerateWalletOptions,
+  Keychains,
+} from '@bitgo/sdk-core';
 import { UnifiedWalletID, UnifiedWallet, GenerateUnifiedWalletOptions } from '../types';
 import { supportedCoins } from './types';
 import { UnifiedWallets } from '../unifiedWallets';
 
 export class EcdsaUnifiedWallets extends UnifiedWallets {
```

Comment:
perhaps `EcdsaEVMUnifiedWallets` name would be more approriate? since looks like we are fixing this to be EVM specific